File: weisi/src/main/java/com/cjq/lib/weisi/communicator/receiver/ReceiveException.java
package com.cjq.lib.weisi.communicator.receiver;

import java.io.IOException;

/**
 * Created by CJQ on 2018/3/12.
 */

public class ReceiveException extends IOException {

    private static final long serialVersionUID = -1801596232075676570L;

    public ReceiveException() {
        super();
    }

    public ReceiveException(String message) {
        super(message);
    }

    public ReceiveException(String message, Throwable cause) {
        super(message, cause);
    }

    public ReceiveException(Throwable cause) {
        super(cause);
    }
}

File: wsb/src/main/java/com/wsn/lib/wsb/communicator/receiver/ReceiveException.kt
package com.wsn.lib.wsb.communicator.receiver

import java.io.IOException

/**
 * Created by CJQ on 2018/3/12.
 */

class ReceiveException : IOException {

    constructor() : super() {}

    constructor(message: String) : super(message) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}

    constructor(cause: Throwable) : super(cause) {}

    companion object {

        private const val serialVersionUID = -1801596232075676570L
    }
}
